File: Problem_01/AC_solution.cpp
#include<bits/stdc++.h>  
using namespace std;

int main() {
    string s;
    string msg = "NO";
    cin >> s;
    for(int i = 0 ; i < s.size(); ++i){
        if(s[i] == 'W')
        {
            msg = "YES";
            break;
        }
    }
    cout << msg;
    return 0;
}

File: Problem_01/AC_solution.java
import java.util.Scanner;

public class AC_solution {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        String s = scanner.next();
        String msg = "NO";

        for (int i = 0; i < N; ++i) {
            if (s.charAt(i) == 'W') {
                msg = "YES";
                break;
            }
        }

        System.out.println(msg);
    }
}
